db/model/Gdoc/Gdoc.ts

```diff
@@ -245,6 +310,9 @@ export class Gdoc extends BaseEntity implements OwidArticleType {
             []
         )
 
+        const chartRecords = await getChartsRecords()
```

Comment:
It's interesting that you use the content graph here, but `Chart.mapSlugsToIds()` above.

I'm not certain they have the exact same behaviour? E.g. with regard to redirects, or unpublished charts?